### ISSUE TITLE
Feature Request: Playback speed through zero and into reverse

### ISSUE THREAD

**Issue by jonheal** [LomasModules/LomasModules#14]
Have I asked for this already in another forum? I feel like I'm repeating myself ... but it would be great if your sample player could take playback speed right to zero and then ramp up in reverse just like it does in forward play.

**Comment by LomasModules** [comment]
I think I have not seen this request. But don't fully understand what you mean sorry. How do you take the speed playback to zero? 
Forward and backwards playing is different only in how looping points works.

**Comment by LomasModules** [comment]
I think I get it.

If what you want is a vinyl like fx nysinthi has a sampler that implements this I think.

I may add the option in the future but I don't have much time rigth now.

**Comment by jonheal** [comment]
Correct. Control over the speed of playback right down to stopping it and then playing it in reverse, like dragging a vinyl record forwards and backwards under the needle.
